standard/table.lua

```diff
@@ -0,0 +1,270 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Table
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+---
+-- @author Vogan for Liquipedia
+--
+-- A number of these functions take inspiration from Penlight: https://github.com/lunarmodules/Penlight
+--
+
+--local Class = require('Module:Class')
+
+local Table = {}
+
+function Table.randomize(tbl)
+	math.randomseed(os.time())
```

Comment:
this doesn't look right - rng should not be seeded at the beginning of a function using randomness.

also os.time() has second precision so this is just resetting the rng with the same seed every time

Reply:
Nice one for a follow-up